File: js/src/Ice/EndpointInfo.js
//
// Copyright (c) ZeroC, Inc. All rights reserved.
//

const Ice = require("../Ice/ModuleRegistry").Ice;
Ice._ModuleRegistry.require(module,
                            [
                                "../Ice/Endpoint"
                            ]).Ice;

let IceSSL = Ice._ModuleRegistry.module("IceSSL");

/**
 * Provides access to an SSL endpoint information.
 *
 **/
IceSSL.EndpointInfo = class extends Ice.EndpointInfo
{
    constructor(underlying, timeout, compress)
    {
        super(underlying, timeout, compress);
    }
};

exports.IceSSL = IceSSL;

File: java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/SSL/EndpointInfo.java
//
// Copyright (c) ZeroC, Inc. All rights reserved.
//

package com.zeroc.Ice.SSL;

/** Provides access to an SSL endpoint information. */
public abstract class EndpointInfo extends com.zeroc.Ice.EndpointInfo {
    public EndpointInfo() {
        super();
    }

    public EndpointInfo(com.zeroc.Ice.EndpointInfo underlying, int timeout, boolean compress) {
        super(underlying, timeout, compress);
    }

    public EndpointInfo clone() {
        return (EndpointInfo) super.clone();
    }
}
